Sources/App/Models/Repository.swift

```diff
@@ -91,26 +94,29 @@ final class Repository: Model, Content {
     @Field(key: "stars")
     var stars: Int?
 
-    @Field(key: "is_in_organization")
-    var isInOrganization: Bool?
-    
     @Field(key: "summary")
     var summary: String?
-    
+
+    @Field(key: "topics")
+    var topics: [String]
+
     // initializers
     
     init() { }
     
     init(id: Id? = nil,
          package: Package,
          authors: [Author] = [],
-         summary: String? = nil,
          commitCount: Int? = nil,
+         defaultBranch: String? = nil,
          firstCommitDate: Date? = nil,
+         forks: Int? = nil,
+         forkedFrom: Repository? = nil,
+         isArchived: Bool? = nil,
+         isInOrganization: Bool? = nil,
```

Comment:
There's quite a bit of commit noise here but I finally twitched and had to sort this long list of parameters.

That's one of the reasons I created a number of branches so we can make this PR smaller.